tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -305,8 +305,19 @@ object TastyFormat {
    *  However, tooling with an experimental TASTy version
    *  is able to read final TASTy documents if the file's
    *  `MinorVersion` is strictly less than the current value.
+   *
+   *  When the format gets fixed for a minor version of the language,
+   *  e.g. at the point of the release of Scala 3.2.0-RC1,
+   *  the experimental version number should be changed to 0 and after that
+   *  the same non-experimental verion of TASTy should be used
+   *  until the snapshot version of the next minor language release
+   *  (3.3.0-RC1-SNAPSHOT in this case) when TASTy minor version gets bumped
+   *  and the experimental version number gets set to a 1.
+   *  The period between Scala 3.3.0-RC1-SNAPSHOT and 3.3.0-RC1
+   *  is the only time when experimental TASTy version number can be incremented.
+   *  When 3.3.0-RC1 gets released the experimental version number should be set to 0 again.
    */
-  final val ExperimentalVersion: Int = 1
+  final val ExperimentalVersion: Int = 0
```

Comment:
If we change this then the nightly will have a stable release version

Reply:
seems like discussion on https://github.com/lampepfl/dotty/issues/14306 seems to agree with doing this however

Reply:
This seems to be a workaround #14306 rather than a fix. How would that solve the exact same problem for 3.2 when and later versions?

Reply:
This seems to indicate that all nightly and snapshots will be tagged as stable. Can't we do something more fine-grained?